packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -170,24 +170,13 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
             }
 
             if (isTimestamp) {
-                const parsedTimestampToUtc = rule.values
-                    ? dayjs(
-                          parseTimestamp(
-                              formatTimestamp(
-                                  rule.values[0],
-                                  TimeFrames.MILLISECOND,
-                              ),
-                              TimeFrames.MILLISECOND,
-                          ),
```

Comment:
This parse timestamp was returning a non-standard string, then the .utc() call on the next line was causing the TZ to be dropped and this time was not correct.